subprojects/gst-docs/examples/tutorials/.vscode/tasks.json

```diff
@@ -0,0 +1,14 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "meson",
+			"mode": "build",
+			"problemMatcher": [
+				"$meson-gcc"
+			],
+			"group": "build",
+			"label": "Meson: Build all targets"
+		}
+	]
+}
```

Comment:
It'd be good if you configure your editor to always put a newline at the end of files :slightly_smiling_face:

Reply:
It also useful to highlight somehow the extra spaces at the end of line

Reply:
I though it was negative to leave an empty line at the end of files... at least that's what I understood for your "Extra empty lines at EOF" corrections a few commits ago